src/projector.ts

```diff
@@ -49,7 +57,10 @@ export interface ProjectorMixin {
 	 * @param tagName If `append` is `true` then `tagName` will be used to determine what tag name
 	 *                is used to append to the root element. Defaults to `div`.
 	 */
+	attach(options?: AttachOptions): Handle;
+	// FIXME Remove old attach() overload
 	attach(append?: boolean, tagName?: string): Handle;
+	attach(append?: boolean | AttachOptions, tagName?: string): Handle;
```

Comment:
Go ahead and break this API backwards... just go with `options?: AttachOptions` as the signature.